recipes/pyopcode/meta.yml

```diff
@@ -0,0 +1,51 @@
+{% set version = "0.2.0" %}
+
+package:
+  name: pyopcode
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/ClinicalGraphics/pyopcode
+  git_tag: {{ version }}
+  
+build:
+  features:
+  - vc9                [win and py27]
+  - vc10               [win and py34]
+  - vc14               [win and py35]
+  number: 0
+
+requirements:
+  build:
+  - python
+  - numpy 1.10*
+  - gcc 4.8.2 [linux]
```

Comment:
Why are we using the `gcc` package? We provide the same compiler for Linux automatically.